web/style/products.scss

```diff
@@ -0,0 +1,130 @@
+@import '_variables.scss';
+@import 'common.scss';
+.category-list {
+    width: 25%;
+    padding: 3px 12px 3px 12px;
+    background-color: #eaeaea;
+    margin-top: -10px;
+    display: none;
+  
+    @include break-point(tablet) {

```

Comment:
Media queries should not be placed within selectors as this leads to duplicate media queries. Instead should be having media query blocks at the end of the module so that you can group the selectors